src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -156,6 +156,11 @@ private Runtime() {}
      * <p> The {@link System#exit(int) System.exit} method is the
      * conventional and convenient means of invoking this method.
      *
+     * @implNote
+     * If the {@link System.Logger#getLogger(String) System.Logger.getLogger("java.lang.Runtime")}
```

Comment:
The link looks wrong to me:
```suggestion
     * If the {@link System#getLogger(String) System.getLogger("java.lang.Runtime")}
```
See https://docs.oracle.com/en/java/javase/19/docs/api/java.base/java/lang/System.html#getLogger(java.lang.String)

Reply:
yep, will fix.  Oddly, the link worked. Thanks